recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
```

Comment:
`requests` in `run` was removed, see https://github.com/bernii/querystring-parser/pull/32/files

Reply:
Got it! Removed